SitecoreDeployer/buildtask/functions-module-v1.psm1

```diff
@@ -195,15 +195,15 @@ function CollectWebAppOutboundIpAddresses{
 
     if (!$resourcePresenceChecked) {
         #get all resrouces in current resource group, and check if resource is present there
-        $webAppResource = (Find-AzureRmResource -ResourceGroupNameContains $resourceGroupName).where({$_.Name -eq "$webAppName" -And $_.ResourceGroupName -eq "$resourceGroupName"})
+        $webAppResource = (Get-AzureRmResource -ResourceGroupName *$resourceGroupName*).where({$_.Name -eq "$webAppName" -And $_.ResourceGroupName -eq "$resourceGroupName"})
         #measure found amount and if less or equal to 0 - we could not find web app
         if (($webAppResource | Measure-Object).Count -le 0) {
             Write-Host "##vso[task.logissue type=warning;] CollectWebAppOutboundIpAddresses: Could not find web app $webAppName in resource group $resourceGroupName. Returning back"
             return;
         }
     }
 
-    $WebAppConfig = (Get-AzureRmResource -ResourceType Microsoft.Web/sites -ResourceName $webAppName -ResourceGroupName $resourceGroupName -ApiVersion $APIVersion)
+    $WebAppConfig = (Get-AzureRmResource -ResourceType Microsoft.Web/sites -Name $webAppName -ResourceGroupName $resourceGroupName -ApiVersion $APIVersion)

```

Comment:
Please, leave ```ResourceName``` here to maintain backward compatibility with older powershell versions (see https://docs.microsoft.com/en-us/powershell/module/azurerm.resources/get-azurermresource?view=azurermps-6.13.0 -  ```ResourceName``` is alias for ```-Name```, while in Powershell v.5 and lower, there no ```-Name``` at all